### DISCUSSION TITLE
Maintenance and optimization of our CI

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#11365] [General]
The user @xmichele is interested in helping us with our CI. I proposed using a discussion to introduce himself and chat about our CI, its current state and improvement proposals.

It seems only team members can start a new discussion so I opened this discussion as a starting point for @xmichele to reply.

**Comment by xmichele** [comment]
Hello everyone!
I'm a devOps engineer and I can give an help especially on Jenkins, Docker, CI/CD, Kubernetes topics.
I started working on this PR with 2 commits https://github.com/cgeo/cgeo/pull/11329
In the next days I'll try to give  a look to the current CI or maybe you already have some interesting points to improve

**Comment by Lineflyer** [comment]
While I have only very limited knowledge about Jenkins, I think you might get some ideas be looking at issues with label "CI server / Build tools" as listed here:
https://github.com/cgeo/cgeo/issues?q=is%3Aopen+is%3Aissue+label%3A%22CI+server+%2F+Build+tools%22

I do however not know which of those are still relevant and/or important.
